src/core/src/package_managers/UbuntuProClient.py

```diff
@@ -0,0 +1,77 @@
+# Copyright 2023 Microsoft Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Requires Python 3.5+
+
+"""This is the Ubuntu Pro Client implementation"""
+
+
+class UbuntuProClient:
+    def __init__(self, env_layer, composite_logger):
+        self.env_layer = env_layer
+        self.composite_logger = composite_logger
+        self.ubuntu_pro_client_install_cmd = 'sudo apt-get install ubuntu-advantage-tools -y'
+
+    def install_or_update_pro(self):
+        """install/update pro(ubuntu-advantage-tools) to the latest version"""
+        try:
+            # Install Ubuntu Pro Client.
+            code, output = self.env_layer.run_command_output(self.ubuntu_pro_client_install_cmd, False, False)
+            if code == 0:
+                self.composite_logger.log_debug("Ubuntu Pro Client installation successful. " + output)
+                return True
+            else:
+                self.composite_logger.log_debug("Ubuntu Pro Client installation failed. " + output)
```

Comment:
Do we need to add the output here to error details within status blob?

Reply:
yes. This need not be in status blob. I will change this

Reply:
Can we have a separate verbose log for output so that it does not get in status blob but is there in telemetry?